lib/app/views/languages/languages.erb

```diff
@@ -5,6 +5,12 @@
 	<% if active %>
 		<article class="article-block">
 			<div class="article-contents">
+				<% if docs[:docs][:about].empty? == false %>
```

Comment:
Since we have a presenter for this, let's go ahead and hide the hash access stuff. Maybe: `docs.about`

Also, instead of comparing to `== false` let's use `!` infront of the boolean, (`!docs.about.empty?`)